### PR TITLE
vendor neutrality in README for VS Code and VSCodium

### PR DESCRIPTION
**Pull request by palmskog**
Since VsCoq works on both VS Code+marketplace and VSCodium+VSX, this PR makes the README neutral on the choice between these two vendoring options.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,31 +42,31 @@ original author of this extension is @siegebell.
...
+The recommended way to install VsCoq is via the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=maximedenes.vscoq) or [Open VSX](https://open-vsx.org/extension/maximedenes/vscoq).
 
 ## Screenshots
 ![vscoq](https://user-images.githubusercontent.com/647105/64025392-dbf12100-cb3c-11e9-8e7f-5c63296500f9.png)
 
 ## Instructions
 1. install [Coq](https://coq.inria.fr/download)
-2. install [vscode](https://code.visualstudio.com/)
-3. run `code`
+2. install [VsCode](https://code.visualstudio.com/) or [VsCodium](https://vscodium.com/)
```
Comment (TheoWinterhalter) [inline README.md:56]: I don't `VsCode` is a standard way to write. It seems it should either be `vscode` or `VS Code` (see e.g. https://code.visualstudio.com/blogs/2022/05/18/dev-container-cli or other places on the website where they use `VS Code`).

Comment (palmskog) [reply README.md]: I switched to `VS Code` everywhere, abbreviating `Visual Studio Code`.

**Comment by Alizter** [comment]
Is it VsCoq or VSCoq? Is it VS Code, VSCode or VsCode?

**Comment by Zimmi48** [comment]
It's VS Code, VSCodium and, for now, at least, VsCoq (according to the README). So Karl's modifications are correct AFAICT.

**Comment by palmskog** [comment]
@huynhtrankhanh please consider reviewing (and possibly merging) this PR

**Pull request merged**